packages/core/src/components/graphs/bar-grouped.ts

```diff
@@ -316,15 +320,34 @@ export class GroupedBar extends Bar {
 		return this.getBarWidth() * activeData.length + totalGroupPadding;
 	}
 
+	protected getDomainScaleStep() {
+		const domainScale = this.services.cartesianScales.getDomainScale();
+		const activeData = this.model.getGroupedData(this.configs.groups);
+
+		let step = 70;
```

Comment:
Should add a comment for what step is & why it is initialized to 70.

Considering `70` is also referenced in line 350, you can even create a readonly variable and assign it to step 🤔. Thoughts @theiliad on using `readonly` modifiers? It isn't used anywhere in the project

Reply:
Other than adding a comment, LGTM!